### ISSUE TITLE
[BUG] Better idempotency of the role

### ISSUE THREAD

**Issue by ulvida** [UdelaRInterior/ansible-backuppc-client#4]
[This task](https://github.com/UdelaRInterior/ansible-backuppc-client/blob/master/tasks/ssh.yml#L27) and the following, that remove existing known IPs associated to de ssh key, where introduced to address the case when a host is re-installed. 
However, they make the role not idempotent, as they remove such lines of known_hosts file, even if they are the same than configured just after.
We should search a way to check first if the key associated to the IPs are correct or no.

**Comment by Dkmarce** [comment]
why not get the key first with `ssh-keyscan` and then update the `known_hosts`?

Reading [this](https://stackoverflow.com/a/39083724) makes me wonder if the module just update the key.

```
- name: Add/update the public key in the '{{ ssh_known_hosts_file }}'
    known_hosts:
      name: "{{ item.item }}"
      key: "{{ item.stdout }}"
      path: "{{ ssh_known_hosts_file }}"
    with_items: "{{ ssh_known_host_results.results }}"
```

worth a try maybe.

**Comment by ulvida** [comment]
Interesting reference! It seems indeed that the Ansible `known_hosts` module can update the key. the problem of the previous version then, was that it only considered the domain name, not the IPs of the client hosts. 
I'll try to use it for the bug purpose.

**Comment by ulvida** [comment]
Coded and tested! I just delet IPs keys, and add or update the host key. The IP keys are added without confirmation when backuppc runs ssh. 
Solved with [v2.1.1](https://github.com/UdelaRInterior/ansible-backuppc-client/releases/tag/v2.1.1)
